changelogs/fragments/1043-bug-title-zos_operator-is-passing-wrong-value-to-zoauopercmd.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - zos_operator - The module was ignoring the wait time argument. The module now passes the wait time argument to zoau.
+    (https://github.com/ansible-collections/ibm_zos_core/pull/1044).
+
+  - zos_operator_action_query - The module was ignoring the wait time argument. The module now passes the wait time argument to zoau.
+    (https://github.com/ansible-collections/ibm_zos_core/pull/1044).
```

Comment:
Everything looks good, I would just say that it would look better to write `ZOAU` in all caps.

Reply:
I agree.